setup_simulations/py/read_gadget.py

```diff
@@ -23,7 +23,7 @@ def _check_gadget_config(config):
     config.validate(vtor)
     # Check unsupported configurations
     if config['MNue']+config['MNum']+config['MNut'] > 0.:
-        raise ValueError("Simulations should not have neutrions")
+        print("Sim has neutrinos")
```

Comment:
Is this safe now? If so, we should just delete the warning. I think we added in the beginning while developing, since we hadn't tested that the code would work for massive neutrinos, but it does seem like the code properly reads the right cosmology even with massive neutrinos